setup.py

```diff
@@ -179,10 +179,10 @@ def build_extension(self, ext: CMakeExtension):
 
 suffix = backend.replace("lightning_", "")
 suffix = suffix[0].upper() + suffix[1:]
-pennylane_plugins = [f"{device_name} = pennylane_lightning:Lightning{suffix}"]
-
+pennylane_plugins = [f"lightning.qubit = pennylane_lightning:LightningQubit",
+                     f"lightning.kokkos = pennylane_lightning:LightningKokkos"]
```

Comment:
As a response to my earlier question about deleting after installing, it may be required to separate these into

```
pennylane_plugins = [f"lightning.qubit = pennylane_lightning_qubit:LightningQubit",
                     f"lightning.kokkos = pennylane_lightning_kokkos:LightningKokkos"]
```
directories, so that they can be tracked separately in their respective manifests.